### DISCUSSION TITLE
Configuration hot reload

### DISCUSSION THREAD

**Discussion by NN-Binary** [wiresock/proxifyre#25] [Ideas]
Cool tool! Thank you

is there a way of editing the config file without having to restart the program?

**Comment by wiresock** [comment]
The current build lacks a hot reload feature. Creating a WCF-based tool to implement this is possible and I will add it to the TODO list. However, I cannot commit to a specific timeline due to my limited availability to work on it.

**Reply by LiMoon** [reply]
hi, 2025 Is support currently available?

**Comment by NN-Binary** [comment]
Thank you for your work!
